CHANGELOG.rst

```diff
@@ -5,10 +5,51 @@ This document describes changes between each past release as well as
 the version control of each dependency.
 
 
-4.5.0 (unreleased)
+4.5.0 (2017-11-16)
 ==================
 
-- Nothing changed yet.
+kinto
+'''''
+
+**kinto 7.5.1 → 7.6.0**: https://github.com/Kinto/kinto/releases/tag/7.6.0
+
+**Protocol**
+
+- When a record is pushed with an older timestamp, the collection
+  timestamps is not bumped anymore. (Kinto/kinto#1361)
+
+**New features**
+
+- A new custom logging formatter is available in ``kinto.core``. It fixes the issues of
+  `mozilla-cloud-services-logger <https://github.com/mozilla/mozilla-cloud-services-logger>`_.
+  Consider migrating your logging settings to :
+
+::
+
+    [formatter_json]
+    class = kinto.core.JsonLogFormatter
```

Comment:
Maybe we should take advantage of this PR to update the logging config in the example.ini

Reply:
Can I let you update it?